src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -215,7 +221,16 @@ const styles = StyleSheet.create({
     fontSize: 14,
     fontStyle: 'normal',
     fontWeight: Platform.OS === 'android' ? '700' : '600',
-    lineHeight: 22,
+    // lineHeight: 22,
+    marginBottom: 12
+  },
+  bioTextNull: {
+    color: COLORS.lightgrey,
+    fontFamily: 'Inter',
+    fontSize: 14,
+    fontStyle: 'italic',
+    fontWeight: '400',
+    lineHeight: 20,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out `lineHeight` property in the `bioText` style has been removed, and a new style `bioTextNull` has been added for the default bio message. The new style includes an italic font style and a lighter color, which differentiates it from the actual bio text.
```diff
- lineHeight: 22,
+ // lineHeight: 22,

+ bioTextNull: {
+   color: COLORS.lightgrey,
+   fontFamily: 'Inter',
+   fontSize: 14,
+   fontStyle: 'italic',
+   fontWeight: '400',
+   lineHeight: 20,
+   marginBottom: 12
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->